DayrSuryan/DSEthiop9.xml

```diff
@@ -385,8 +456,19 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                </physDesc>
 
                <history>
-                  <provenance>
-                  </provenance>
+                  <origin>
+                     The Coptic patriarch <persName ref="PRS5714JohnXIX">John XIX</persName> (in tenure 1928-42)
+                     is mentioned in supplications, together with Ṗeṭros (<locus target="#5v"/>, possibly referring to the metropolitan 
+                     <persName ref="PRS7830Petros">Ṗeṭros IV</persName> of Ethiopia (in tenure 1882-1917) ,
+                     and <persName ref="PRS8033Qerellos">Qerǝllos</persName> of Ethiopia (in tenure 1929-50) (<locus target="#124r"/>).
+                     The manuscript was likely produced during <persName ref="PRS5714JohnXIX"/>'s tenure
+                        <origDate notBefore="1928" notAfter="1942">1928-42</origDate>. The reference to
+                     <persName ref="PRS7830Petros"/> does not fit within this period. 
+                     <persName ref="PRS11960HabtaM"/> is mentioned in supplications and was probably the original owner of the manuscript. He
+                     probably donated the manuscript to <persName ref="PRS12002Gabrami"/>. Both are mentioned in <ref target="#a1">Additio 1</ref>
+                     together with the monasteries of <placeName ref="INS0352EA"/> and
+                     <placeName ref="INS0354DB"/>.
```

Comment:
The Coptic patriarch <persName ref="PRS5714JohnXIX">John XIX</persName> (in tenure 1928-42)
 is mentioned in supplications, together with <persName ref="PRS8033Qerellos">Qerǝllos</persName> of Ethiopia (in tenure 1929-50) (<locus target="#124r"/>). The manuscript was likely produced sometime in <origDate notBefore="1928" notAfter="1942">1928-42</origDate>.  The identity of Ṗeṭros (<locus target="#5v"/>, also mentioned in the supplications, is unclear. <persName ref="PRS11960HabtaM"/>  was probably the original owner of the manuscript; he donated the it to <persName ref="PRS12002Gabrami"/>. Both are mentioned in <ref target="#a1">Additio 1</ref>  together with the monasteries of <placeName ref="INS0352EA"/> and
                     <placeName ref="INS0354DB"/>.

I suggest this above. Petros is unlikely the metropolitan; there were two more hierarchs by this name but we will not speculate. Please indicate their affiliations, this is very interesting.

Reply:
new proposal, more confusion. Petros and Qerellos are both referred to as "bəḍuʿ ̆pāpāsna" in the manuscript